unified-prototype/unified-plugin/plugin-common/src/main/java/org/gradle/api/experimental/common/ApplicationDependencies.java

```diff
@@ -0,0 +1,18 @@
+package org.gradle.api.experimental.common;
+
+import org.gradle.api.artifacts.dsl.DependencyCollector;
+import org.gradle.api.artifacts.dsl.GradleDependencies;
+import org.gradle.api.plugins.jvm.PlatformDependencyModifiers;
+import org.gradle.api.plugins.jvm.TestFixturesDependencyModifiers;
+import org.gradle.declarative.dsl.model.annotations.Restricted;
+
+/**
+ * The declarative dependencies DSL block for an application.
+ */
+@SuppressWarnings("UnstableApiUsage")
+@Restricted
+public interface ApplicationDependencies extends PlatformDependencyModifiers, TestFixturesDependencyModifiers, GradleDependencies {
```

Comment:
🤔 I think we should just extend `Dependencies` directly until we get the modifiers from these other types working. 

Also on the `LibraryDependencies`.